lib/code_corps/ember_client.ex

```diff
@@ -0,0 +1,32 @@
+defmodule CodeCorps.EmberClient do
+  @moduledoc ~S"""
+  Confirms URLs for the Ember client app routes
+  """
```

Comment:
I wouldn't reference Ember specifically here, but instead just call this something like `WebClient`.